JDStatusBarNotification/JDStatusBarView.m

```diff
@@ -7,6 +7,9 @@
 //
 
 #import "JDStatusBarView.h"
+#import <sys/utsname.h>
+
+CGFloat const ADJUSTIPHONEX = 29;
```

Comment:
same here, can we find a solution without magic numbers?

Reply:
I can get the top constraint 44 of the safe area, but the result is not consistent with the original design (too high), so I have to manually adjust the height value.